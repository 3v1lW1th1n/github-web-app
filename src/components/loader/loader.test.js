import 'jest-styled-components'

import React from 'react'
import { shallow } from 'enzyme'
import Loader from './'

describe('<Loader />', () => {
  it('should match default snapshot', () => {
    const component = shallow(<Loader />)
    expect(component).toMatchSnapshot()
  })

  it('should have the right displayName', () => {
    expect(Loader.displayName).toEqual('Loader')
  })
})
