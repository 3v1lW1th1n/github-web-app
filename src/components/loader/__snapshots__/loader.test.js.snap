// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> should match default snapshot 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "Loader-sc-1fvxvf6-0",
        "isStatic": true,
        "rules": Array [
          "display:inline-block;position:relative;width:100%;display:flex;align-items:center;justify-content:center;height:calc(100vh - 64px);div{box-sizing:border-box;display:block;position:absolute;width:81px;height:81px;margin:6px;border:6px solid #0095B6;border-radius:50%;animation:lds-ring 1.2s cubic-bezier(0.5,0,0.5,1) infinite;border-color:#0095B6 transparent transparent transparent;}div:nth-child(1){animation-delay:-0.45s;}div:nth-child(2){animation-delay:-0.3s;}div:nth-child(3){animation-delay:-0.15s;}@keyframes lds-ring{0%{transform:rotate(0deg);}100%{transform:rotate(360deg);}}",
        ],
      },
      "displayName": "Loader",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "Loader-sc-1fvxvf6-0",
      "target": [Function],
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
